_sass/_menu-item.scss

```diff
@@ -0,0 +1,52 @@
+.menu {
+  margin: 0;
+  padding: 0;
+}
+
+.menu-item {
+  justify-content: space-between;
+  border-bottom: 1px solid $border-color;
+  list-style: none;
+}
+
+.menu-link {
+  --_menu-link-background-color: 
+    var(--menu-link-background-color, #{$white});
+  --_menu-link-background-color--hover: 
+    var(--menu-link-background-color--hover, #{mix(white, $cyan, 10%)});
+  
+  display: flex;
+  align-items: center;
+  width: 100%;
+  gap: 0.25rem;
+  padding: 0.5rem 1rem;
+  
+  background-color: var(--_menu-link-background-color);
+  color: $navy;
+  
+  text-decoration: none;
+
+  &.icon {
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+
+  &:hover {
+    background-color: var(--_menu-link-background-color--hover);
+  }
+
+  &.cyan {
+    --_menu-link-background-color: #{$cyan};
+    --_menu-link-background-color--hover: #{mix(white, $cyan, 25%)};
+  }
+
+  &.yellow {
+    --_menu-link-background-color: #{$yellow};
+    --_menu-link-background-color--hover: #{mix(white, $yellow, 25%)};
+  }
+
+  &.light-blue {
+    --_menu-link-background-color: #{$cyan-a-10};
+    --_menu-link-background-color--hover: #{mix(white, $cyan, 25%)};
+  }
+}
```

Comment:
@rpeterman-gp Got the variables set for this menu.

Reply:
Oops. I goofed on which custom property should be change in the color variations. Should be setting the version w/o the underscore. I will fix the button styles to demonstrate.